bundlegen/core/bundle_processor.py

```diff
@@ -792,11 +804,11 @@ def _process_storage(self):
                     # We want to ensure that the same image is used if we upgrade to a new bundle version to persist
                     # app data across upgrades. This will ensure the filename is always the same providing the destination
                     # path (path inside the container) do not change.
-                    hash_key = dest_path.encode('ascii')
-                    img_name = sha256(hash_key).hexdigest()
+                    
+                    
 
                     source_path = os.path.join(
-                        persistent_storage_dir, self.app_metadata['id'], f"{img_name}.img")
+                        persistent_storage_dir, self.app_metadata['id'], f"{Utils.get_random_string(8)}.img")
```

Comment:
what's the point of this change ?

Reply:
I have mistakenly updated the code hence reverting to previous one.